bucket/pencil.json

```diff
@@ -1,22 +1,22 @@
 {
-    "version": "3.0.4",
+    "version": "3.1.0",
     "description": "GUI prototyping tool.",
-    "homepage": "https://pencil.evolus.vn/",
+    "homepage": "https://pencil.evolus.vn",
     "license": {
-        "identifier": "GPL-2.0-only|Freeware",
+        "identifier": "GPL-2.0-only",
```

Comment:
It would be nice to mention that commercial license.

I do not know how extractly, but it could be `Custom`.

Reply:
I just thought that it makes no sense for end users. Should we add another identifier?

Reply:
Then you could just keep license as string since it is valid SPDX